src/app/market/shared/orders/orders.component.ts

```diff
@@ -53,16 +57,45 @@ export class OrdersComponent implements OnInit {
     this.profileService.default().take(1).subscribe(
       profile => {
         this.profile = profile;
-        this.loadOrders();
+        this.callLoadOffers();
       });
   }
 
-  loadOrders(): void {
-    this.bid.search(this.profile.address, this.type).take(1).subscribe(orders => {
-      console.log('called >>>>>>>>>>>>>>>>>', orders);
-      orders.orders.reverse();
-      this.orders = orders;
+  callLoadOffers() {
+    this.loadOrders()
+    this.timer = Observable.interval(1000 * 10);
```

Comment:
This is every 10s, not every 30s

Reply:
@kewde  yes, 30 sec seems like a little late, so I changed it to 10 sec. :)